src/google_takeout_metadata/exif_writer.py

```
@@ -26,85 +26,236 @@ def build_exiftool_args(meta: SidecarData, image_path: Path | None = None, use_l
+        album_args = []
+        for album in meta.albums:
+            album_keyword = f"Album: {album}"
+            album_args += [
+                f"-XMP-dc:Subject+={album_keyword}",
+                f"-IPTC:Keywords+={album_keyword}",
+            ]
+        _run_exiftool_command(image_path, album_args, append_only=True)
+    
+    # 4. Write rating only if it doesn't exist (conditional)
+    if meta.favorite:
+        rating_args = ["-if", "not $XMP:Rating", f"-XMP:Rating=5"]
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)